Getting Started rework (#4629)
wazuh/wazuh-documentation: source/getting-started/components/kibana.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. _components_kibana:

Kibana
======

> Kibana is a flexible and intuitive web interface for mining, analyzing, and visualizing data. In combination with the Wazuh Kibana plugin, it is also used for management and monitoring of the Wazuh infrastructure. From the Wazuh user interface, identity and access management can be controlled as well, creating and managing users, and assigning roles as needed. 

Reviewer: ```suggestion
Kibana is a flexible and intuitive web interface for mining, analyzing, and visualizing data. In combination with the Wazuh Kibana plugin, it is also used for the management and monitoring of the Wazuh infrastructure. From the Wazuh user interface, you can create and manage users, and assign roles as needed. In addition, identity and access management can be controlled from the user interface as well.
```

Data visualization, mining, and analysis
----------------------------------------

> You can search alerts classified by modules and filter them using the different views. Users are be able to explore the alerts at the Wazuh cluster level and on a specific agent. Wazuh provides out-of-the-box dashboards, allowing you to seamlessly navigate through the user interface. Users can quickly visualize security events, detected vulnerable applications, file integrity monitoring data, configuration assessment results, cloud infrastructure monitoring events, and regulatory compliance, such as PCI DSS, GDPR, CIS, HIPAA, and NIST 800-53 standards. It can also be used for other data visibility purposes.

Reviewer: I have concerns as regards this sentence:

> You can search alerts **classified by modules** and filter them using the different views.

Can you check its accuracy and modify it if needed?

      <div class="col-12 col-lg-6">

.. thumbnail::  ../../images/getting_started/module_compliance.png 
    :align: center
    :title: Modules directory

.. raw:: html

  </div>
  </div>
  </div>
  </div>
   </div>
   </div>

Monitoring deployed agents
--------------------------

From the web interface you manage your agents, their configuration, and data inventory. Inspecting the status, statistics, and health of any agent deployed to your system is streamlined with filters for fine-tuning and quick access to data. You can also create deployment commands to install and configure new Wazuh agents. 

Additionally, agent groups and centralized configuration can be managed and edited through the interface as well.

.. thumbnail::  ../../images/getting_started/dashboard_screenshot_agent.png 
      :align: center
      :title: Agent dashboard overview

  

Management of Wazuh infrastructure
----------------------------------

Kibana runs on top of the indexed data in an Elasticsearch cluster and provides you with a directory dedicated to administrate and monitor your infrastructure. From there, you can manage and configure your Wazuh cluster rules and decoders, and CDB lists. 

> Data analysis visualization gives you full visibility into your cluster infrastructure for fast threat detection and remediation. Users can check the status, logs, and statistics of Wazuh clusters and create their own downloadable reports using predefined templates, or personalize their own if required. Wazuh provides the tools for comprehensive and streamlined security analytics.

Reviewer: ```suggestion
Data analysis visualization gives you full visibility into your cluster infrastructure for fast threat detection and remediation. Users can check the status, logs, and statistics of Wazuh clusters and create their own downloadable reports using predefined templates, or personalize their own if required. Wazuh provides the tools for comprehensive and efficient security analytics.
```

.. raw:: html
    
    <div class="images-rn-420-container">
    <div class="images-rn-420">

.. thumbnail::  ../../images/getting_started/dashboard_administration.png 
      :align: center
      :title: Administration

.. thumbnail::  ../../images/getting_started/dashboard_status.png 
      :align: center
      :title: Status and reports

.. raw:: html

    </div> 

Dev tools
---------

User can easily access developer tools integrated directly into the Wazuh Kibana plugin. 

The Ruleset Test tool provides an input box for reading sample logs and an output box to visualize the test results. This feature of the Wazuh Kibana plugin allows you to test sample logs directly on the web user interface and see how the ruleset reacts to specific log messages.

> The Wazuh Kibana plugin also includes an API console for users to interact with the Wazuh API. This API accommodates complete remote management of the Wazuh infrastructure and, from the API Console, you can easily perform everyday actions such as adding agents, restarting components, creating roles, or looking up syscheck details.

Reviewer: ```suggestion
The Wazuh Kibana plugin also includes an API Console for users to interact with the Wazuh API. This API accommodates complete remote management of the Wazuh infrastructure and, from the API Console, you can easily perform everyday actions such as adding agents, restarting components, creating roles, or looking up syscheck details.
```
